pallets/pools/src/lib.rs

```diff
@@ -290,18 +348,12 @@ pub mod pallet {
 			+ Convert<Self::TrancheWeight, Self::Balance>
 			+ From<u128>;
 
-		/// A fixed-point number which represents the value of
-		/// one currency type in terms of another.
-		type BalanceRatio: Member
-			+ Parameter
-			+ Default
-			+ Copy
-			+ TypeInfo
-			+ FixedPointNumber<Inner = Self::Balance>;
-
-		/// A fixed-point number which represents an
-		/// interest rate.
-		type InterestRate: Member
+		/// A fixed-point number which represents a Self::Balance
+		/// in terms of this fixed-point representation.
+		///
+		/// NOTE: One must be cautious with using One::one() in
```

Comment:
Why must one be cautious with using One::one() in this context?

Reply:
As `One::one().into_inner()` is of type `Balance` but has a random decimal (from the perspective of the pallet. Maybe the doc comment is more misleading than clarifying.

Reply:
> Maybe the doc comment is more misleading than clarifying.
Yeah I think this is the case, I'd either remove it or add the context above.